lib/ui/channel_buffers.dart

```diff
@@ -185,6 +187,28 @@ class ChannelBuffers {
       await callback(message.data, message.callback);
     }
   }
+
+  String _getString(ByteData data) {
+    final ByteBuffer buffer = data.buffer;
+    final Uint8List list = buffer.asUint8List(data.offsetInBytes, data.lengthInBytes);
+    return utf8.decode(list);
+  }
+
+  /// Send a control message.
```

Comment:
Somewhat unexpected that the name here is send a control message. 

Wouldn't the verb be intercept or monitor?

Reply:
switched to "handle"